utils.go

```diff
@@ -101,6 +101,15 @@ func (pp PagePool) Get(create func() *Page) *Page {
 	return p
 }
 
+// TryGet a page from the pool, allow error. Use the [PagePool.Put] to make it reusable later.
+func (pp PagePool) TryGet(create func() (*Page, error)) (*Page, error) {
```

Comment:
I've tried it before, but `MustGet` is restricted to be implemented in `must.go` because of the lint. 
Changing to PagePool.Get also could involve the chrome-extensions. I'll rename it if these changes don't impact a lot.

Reply:
Sure, move the code to must.go file. Keep the `Get` one in the utils.go file.

Reply:
Done